Specs/DateTime/Spanish/DateTimePeriodExtractor.json

```diff
@@ -202,28 +191,6 @@
       }
     ]
   },
-  {
-    "Input": "Regresaré a la noche",
-    "Results": [
-      {
-        "Text": "a la noche",
-        "Type": "datetimerange",
-        "Start": 10,
-        "Length": 10
-      }
-    ]
-  },
-  {
-    "Input": "Regresaré a la madrugada",
```

Comment:
We could change this test to `Regresaré por la madrugada` instead of deleting it.